scripts/generate_all_jobs.py

```diff
@@ -58,6 +58,12 @@ def main(argv=sys.argv[1:]):
     print('')
 
     config = get_index(args.config_url)
+    if args.config_url.startswith('file:'):
+        sys.stderr.write('WARNING: Local file system path used for ',
```

Comment:
You should use `sys.stderr` for this since e.g. now you lack a trailing newline. Instead use: `print('...', file=sys.stderr)`

Also please don't align subsequent line.

Reply:
Thanks. I changed it to ``print('...', file=sys.stderr)`` .
About the subsequent lines, this is the indentation enforced by both flake8 and pylint, so I don't know how to change it without failing more checks.